addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -209,11 +209,13 @@
 					<font>font45</font>
 					<label>$INFO[ListItem.ChannelName]</label>
 				</control>
-				<control type="label">
+				<control type="textbox">
 					<top>60</top>
-					<height>200</height>
+					<height>130</height>
```

Comment:
Just curious, why the change in height?

Reply:
Fixes a bug that was there before, but never got triggered before your PR. The height covers 4 lines, progress is drawn over last line, if 4 items are to be displayed 
and a progress bar => 

![screenshot002](https://user-images.githubusercontent.com/3226626/66316014-26717480-e917-11e9-94e4-e0dd49f08fca.png)

... and moving the bar down a bit is not really an option, because there is another bar, shown for in progress shows, that has text labels below the bar graph. Making heights dynamic according the different scenarios clutters the skin xml code in a way that I decided to keep it simple.

Reply:
Probably we now have one more infolabel and if all are shown it might not match with the progressbar below. Might have been a bug before already.